components/emoji_picker/components/emoji_picker_preview.jsx

```diff
@@ -61,7 +61,7 @@ export default class EmojiPickerPreview extends React.Component {
                             id='emojiPickerAliasesPreview'
                             className='emoji-picker__preview-aliases'
                         >
-                            {':' + aliases[0] + ':'}
+                            {':' + aliases.join(': :') + ':'}
```

Comment:
Feel free to leave this as-is, but it might be clearer what we're doing here if you use `map` to add the colons and then `join` to add the spaces between items.

Reply:
`join()` is far more efficient. If it were a more complex transformation of each item, I'd agree with you.